From 0e8fd25546549e03d215b6be730544ee429261a7 Mon Sep 17 00:00:00 2001
From: vipoll <vipollmail@gmail.com>
Date: Sat, 4 Oct 2025 12:09:59 +0400
Subject: [PATCH] arm64: dts: sm8250-xiaomi-pipa: nanosic: invert sleep&reset,
 use constants instead of hardcoded values

(cherry picked from commit f929300f1e4fa7d58e081f82fecb08b5861102d2)
---
 arch/arm64/boot/dts/qcom/sm8250-xiaomi-pipa.dts | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/qcom/sm8250-xiaomi-pipa.dts b/arch/arm64/boot/dts/qcom/sm8250-xiaomi-pipa.dts
index bb0fab087a8e..4a221a238c72 100644
--- a/arch/arm64/boot/dts/qcom/sm8250-xiaomi-pipa.dts
+++ b/arch/arm64/boot/dts/qcom/sm8250-xiaomi-pipa.dts
@@ -784,10 +784,10 @@ nanosic_803: nanosic803@4c {
 		compatible = "nanosic,803";
 		status = "okay";
 		reg = <0x4c>;
-		reset-gpios = <&tlmm 141 0x00>;
-		sleep-gpios = <&tlmm 155 0x00>;
-		vdd-gpios = <&tlmm 127 0x00>;
-		irq-gpios = <&tlmm 100 0x6001>;
+		reset-gpios = <&tlmm 141 GPIO_ACTIVE_LOW>;
+		sleep-gpios = <&tlmm 155 GPIO_ACTIVE_LOW>;
+		vdd-gpios = <&tlmm 127 GPIO_ACTIVE_HIGH>;
+		irq-gpios = <&tlmm 100 IRQ_TYPE_EDGE_RISING>;
 		touchpad-resolution-x = <2879>;
 		touchpad-resolution-y = <1799>;
 		vdd_1v8-supply = <&vreg_l4c_1p8>;
-- 
2.54.0

